src/components/EditorsColumn.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
+import includes from 'lodash/includes';
+import partial from 'lodash/partial';
+
+import EditorContainer from './EditorContainer';
+import Editor from './Editor';
+
+class EditorsColumn extends React.Component {
+  _allErrorsFor(language) {
+    if (language === 'javascript') {
+      return this.props.errors.javascript.items.
+      concat(this.props.runtimeErrors);
```

Comment:
Nitpick but project style has this indented one level deeper than previous.

Also, probably a battle for another day, but this method has always felt bolted-on. Maybe something I can handle in the upcoming `reselect` refactor…